relay-server/src/services/processor.rs

```diff
@@ -3475,10 +3475,10 @@ mod tests {
             processor.handle_process_metrics(&mut token, message);
 
             let value = project_cache_rx.recv().await.unwrap();
-            let ProjectCache::MergeBuckets(merge_buckets) = value else {
+            let ProjectCache::AddMetricBuckets(add_metric_buckets) = value else {
```

Comment:
Had to fix these tests for the PR to pass CI. The failures are related to: https://github.com/getsentry/relay/commit/fc1c2aa9ef2be4562f1062ec512e4c9b96e51118